pkg/resource/scaling_policy/custom_api.go

```diff
@@ -19,8 +19,13 @@ import (
 	"time"
 )
 
+// customSetOutputUpdate sets the LastModifiedTime field to the creationTime
+func (rm *resourceManager) customSetLastModifiedTimeToCreationTime(ko *svcapitypes.ScalingPolicy) {
+	ko.Status.LastModifiedTime = ko.Status.CreationTime
```

Comment:
using this code directly in generator config should work but ok either way

Reply:
prefer the current code for modularity and cleanliness, so keeping as is.